### PR TITLE
WDT-476-documentation

### PR DESCRIPTION
**Pull request by CarolynRountree**
insert a section for the tool.properties properties

### CODE REVIEW

File: site/tool_configuration.md [hunk 1]
```diff
@@ -1,10 +1,26 @@
 ## Tools Configuration
+ - [Tool Property File](#tool-properties)
  - [Model Filters](#model-filters)
  - [Target Environments](config/target_env.md)
  - [Type Definitions](#domain-type-definitions)
  - [Variable Injection](variable_injection.md)
  - [Custom Configuration](#custom-configuration)
 
+ ### Tool Properties
+ The WebLogic Deploy Tooling tools can be configured or tuned from the tool property file. This property file is installed as `<weblogic-deploy>/lib/tool.properties`. You may tune any of the properties in this file or override the file completely by providing your own tool.properties file. Reference the directory where the tool.properties resides using the  `CUSTOM_CONFIG_VARIABLE` Environment variable.
```
Comment (rakillen) [inline site/tool_configuration.md:10]: The environment variable is WDT_CUSTOM_CONFIG. Other sections have this blurb:
"Another option is to configure . . . in a [Custom Configuration](#custom-configuration) directory. Create the `. . .` file in the `$WDT_CUSTOM_CONFIG` directory."

Also add Tools Configuration to the list in the Custom Configurations section.

File: site/tool_configuration.md [hunk 2]
```diff
@@ -1,10 +1,26 @@
 ## Tools Configuration
+ - [Tool Property File](#tool-properties)
  - [Model Filters](#model-filters)
  - [Target Environments](config/target_env.md)
  - [Type Definitions](#domain-type-definitions)
  - [Variable Injection](variable_injection.md)
  - [Custom Configuration](#custom-configuration)
 
+ ### Tool Properties
```
Comment (rosemarymarano) [inline site/tool_configuration.md:9]: Is there a reason why you used the title, "Tool Property File" in the page TOC versus the heading "Tool Properties" in the section? If not, I suggest that you make them the same.

File: site/tool_configuration.md (hunk 1 above)
Comment (rosemarymarano) [inline site/tool_configuration.md:10]: To change this sentence to active voice, "You can configure or tune the WebLogic Deploy Tooling tools using the tool property file."
tool.properties file. -> `tool.properties` file. (code font for a file name; globally)
where the tool.properties resides -> where the `tool.properties` file resides
Environment  -> environment

File: site/tool_configuration.md [hunk 3]
```diff
@@ -1,10 +1,26 @@
...
+ The WebLogic Deploy Tooling tools can be configured or tuned from the tool property file. This property file is installed as `<weblogic-deploy>/lib/tool.properties`. You may tune any of the properties in this file or override the file completely by providing your own tool.properties file. Reference the directory where the tool.properties resides using the  `CUSTOM_CONFIG_VARIABLE` Environment variable.
+ 
+ If a property is removed from the file, or a property value is incorrectly formatted, a `WARNING` message is logged and an internal default value used instead of the bad value.
+ 
+ | Property | Usage |
+ | -------- | ----- |
+ | connect.timeout | The number of milliseconds that WLST waits for the online connect command to complete. A value of 0 means the operation will not timeout. |
```
Comment (rosemarymarano) [inline site/tool_configuration.md:16]: connect.timeout -> `connect.timeout` (property names in code font; globally)
connect -> `connect ` (commands in code font)

File: site/tool_configuration.md [hunk 4]
```diff
@@ -1,10 +1,26 @@
...
+ ### Tool Properties
+ The WebLogic Deploy Tooling tools can be configured or tuned from the tool property file. This property file is installed as `<weblogic-deploy>/lib/tool.properties`. You may tune any of the properties in this file or override the file completely by providing your own tool.properties file. Reference the directory where the tool.properties resides using the  `CUSTOM_CONFIG_VARIABLE` Environment variable.
+ 
+ If a property is removed from the file, or a property value is incorrectly formatted, a `WARNING` message is logged and an internal default value used instead of the bad value.
+ 
+ | Property | Usage |
+ | -------- | ----- |
+ | connect.timeout | The number of milliseconds that WLST waits for the online connect command to complete. A value of 0 means the operation will not timeout. |        
+ | activate.timeout | The number of milliseconds that WLST waits for the activation of configuration changes to complete. A value of -1 means the operation will not timeout. |
+ | deploy.timeout | The number of milliseconds that WLST waits for the un-deployment process to complete. A value of 0 means the operation will not timeout. |
```
Comment (rosemarymarano) [inline site/tool_configuration.md:18]: un-deployment -> undeployment

File: site/tool_configuration.md [hunk 5]
```diff
@@ -1,10 +1,26 @@
...
  - [Type Definitions](#domain-type-definitions)
  - [Variable Injection](variable_injection.md)
  - [Custom Configuration](#custom-configuration)
 
+ ### Tool Properties
+ The WebLogic Deploy Tooling tools can be configured or tuned from the tool property file. This property file is installed as `<weblogic-deploy>/lib/tool.properties`. You may tune any of the properties in this file or override the file completely by providing your own tool.properties file. Reference the directory where the tool.properties resides using the  `CUSTOM_CONFIG_VARIABLE` Environment variable.
+ 
+ If a property is removed from the file, or a property value is incorrectly formatted, a `WARNING` message is logged and an internal default value used instead of the bad value.
+ 
+ | Property | Usage |
```
Comment (rosemarymarano) [inline site/tool_configuration.md:14]: Usage -> Description (or Definition) to align with other WIT docs

**Review by rosemarymarano — COMMENTED** [review]
Few edits

**Review by rakillen — APPROVED** [review]

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**